src/Inciter/Transporter.C

```diff
@@ -329,7 +332,7 @@ Transporter::load( uint64_t nelem )
 }
 
 void
-Transporter::partition()
+Transporter::part()
```

Comment:
Yeah, I agree that his is somewhat misleading, because it is really a leftover from when this was not only a necessary but also a sufficient condition for partitioning (which was incorrect).

Timing is not the main job of `Transporter::part()`.

What it really means is that a necessary condition is met and that condition is the mesh read and computing centroids on all PEs (if a geometric partitioner is configured by the user) have been completed. It does not initiate partitioning. I'm not sure what is a good single word for that, so I punted and kept it `part()` which, I agree, is pretty awful.